controllers/keda/kedacontroller_controller.go

```diff
@@ -439,6 +461,28 @@ func (r *KedaControllerReconciler) installController(ctx context.Context, logger
 		return err
 	}
 
+	if runningOnOpenshift && !r.rotatorStarted {
+		err = rotator.AddRotator(r.mgr, &rotator.CertRotator{
+			SecretKey: types.NamespacedName{
+				Namespace: r.resourceNamespace,
+				Name:      grpcClientCertsSecretName,
+			},
+			// These 3 values ^  vv  match the names the KEDA operator uses to generate its certificate
```

Comment:
Just for my understanding what does `values ^  vv` mean?

Reply:
one value above and two below. Dunno if anybody uses that besides me, so I'm happy to write the field names out to make it more clear